tests/e2e/tests/test_sanity.py

```diff
@@ -224,3 +244,30 @@ def test_katib_experiment(self, setup, cluster, region):
             raise AssertionError("Expected K8sApiException Not Found")
         except K8sApiException as e:
             assert "Not Found" == e.reason
+
+    def test_kserve_with_irsa(self, region, metadata, kfp_client, clone_tensorflow_serving, kserve_iam_service_account, kserve_secret, s3_bucket_with_training_data, kserve_inference_service):
+        #Edit the ConfigMap to change the default domain as per your deployment
+        #kubectl edit configmap config-domain -n knative-serving
+        exec_shell(f"kubectl get configmap config-domain -n knative-serving -o yaml > {KNATIVE_CONFIG_FILE}")
+        subdomain = metadata.get("lb_deps").get("config").get("route53").get("subDomain").get("name")
+        knative_config = load_yaml_file(KNATIVE_CONFIG_FILE)
+        knative_config["data"][subdomain] = ""
```

Comment:
why are we setting the subdomain to empty string?

Reply:
Please read this section https://awslabs.github.io/kubeflow-manifests/docs/component-guides/kserve/tutorial/#configure-a-default-domain-with-knative-serving